target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz3370-rev2.dtsi

```diff
@@ -211,6 +211,7 @@
 		#size-cells = <2>;
 		#address-cells = <3>;
 		device_type = "pci";
+		ranges;
```

Comment:
I think you have to provide an item too, see:
```
  ranges:
    description:
      As described in IEEE Std 1275-1994, but must provide at least a
      definition of non-prefetchable memory. One or both of prefetchable Memory
      and IO Space may also be provided.
    minItems: 1
    maxItems: 3
```
https://www.kernel.org/doc/Documentation/devicetree/bindings/pci/host-generic-pci.txt

I am not exactly sure how this works. 

The PCIe controller uses a 1:1 mapping.

Reply:
This node is a PCIe bridge, not host. I think we can use empty `ranges` here. In `devm_of_pci_get_host_bridge_resources()`, if `ranges` property is empty, this function will directly return a success code. See: https://elixir.bootlin.com/linux/v6.15/source/drivers/pci/of.c#L356

I noticed a lot of `mediatek` target devices' PCIe bridge nodes do not even include `ranges` property. However, DTC doesn't not complain about it because the `device_type = "pci";` is also missing. It seems that the bridge node can be automatically patched. 
https://elixir.bootlin.com/linux/v6.15/source/drivers/pci/of_property.c#L341